pkg/cache/snapshot.go

```diff
@@ -159,7 +165,15 @@ func (c *ClusterQueue) accumulateResources(cohort *Cohort) {
 				cohort.RequestableResources[flvQuotas.Name] = res
 			}
 			for rName, rQuota := range flvQuotas.Resources {
-				res[rName] += rQuota.Nominal
+				// When feature LendingLimit enabled, cohort.RequestableResources indicates
```

Comment:
update the comment in the cohort.RequestableResources field as well.

Reply:
Done.